src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -525,11 +525,11 @@
           - az application-gateway waf-config list-rule-sets
         - name: --disabled-rule-groups
           short-summary: Space separated list of rule groups to disable. This disables the entire group.
-            To disable specifc rules, use '--disabled-rule-ids'.
+            To disable specifc rules, use '--disabled-rules'.
           populator-commands:
           - az application-gateway waf-config list-rule-sets
         - name: --disabled-rules
-          short-summary: Space separated list of rule IDs to disable.
+          short-summary: List of rule IDs to disable, separate arguments not a single space separated string.
```

Comment:
The original help text is correct and is verified by in-place text.  The proposed text would actually be incorrect. Supplying --disabled-rules twice would result in the second overriding the first. 

"Space separated list of ..." is used consistently throughout the CLI in this context.